### ISSUE TITLE
Add function to normalise a data frame

### ISSUE THREAD

**Issue by samueljackson92** [samueljackson92/major-project#41]
Normalise a data frame to unity. Can be done with the aid of sklearn.preprocessing. Also replace this in tSNE code.
